src/operator-docs/filtering/first.ts

```diff
@@ -1,6 +1,90 @@
 import { OperatorDoc } from '../operator.model';
 
 export const first: OperatorDoc = {
-  'name': 'first',
-  'operatorType': 'filtering'
+  name: 'first',
+  operatorType: 'filtering',
+  signature: `public first(predicate: function(value: T, index: number, source: Observable<T>):
+   boolean, resultSelector: function(value: T, index: number): R, defaultValue: R): Observable<T | R>`,
```

Comment:
Do we want to show generic info (ex. Observable<T | R>) within the signature? Some we are, some we aren't. Personally I don't think it adds much value to the end user in this scenario but would be interested to hear others thoughts.

Reply:
Per @benlesh let's leave it out bc it will get hard to read.